### ISSUE TITLE
Did this get merged into semantic-release?

### ISSUE THREAD

**Issue by codeandcats** [semantic-release-plus/semantic-release#73]
Hi @JoA-MoS, I just wanted to know if your commitPaths work got merged into [semantic-release](https://github.com/semantic-release/semantic-release) (doesn't look like it?) or if it got implemented via another means?

(Sorry if a Github Issue is not the best place to ask this)

Cheers.

**Comment by JoA-MoS** [comment]
By this do you mean commit filtering by path?

**Comment by codeandcats** [comment]
Yes. I have a mono repo that includes multiple npm packages that this feature is essential for.

**Comment by JoA-MoS** [comment]
I can connect with the semantic-release team again but they were pretty
clear that this was something they will not consider. See this
https://spectrum.chat/semantic-release/general/commit-filtering-and-other-monorepo-requests~5fc5901a-121e-40e9-b7a2-8a4840fe1341
and this https://github.com/semantic-release/semantic-release/issues/193

**Comment by codeandcats** [comment]
That is a shame. I didn't read the whole thread, but if I got the gist of it right...it sounds like the maintainers are getting held up on not easily being able to completely automate bumping `package A` and update references for `package B` which depends on `A` and bumping it and so on. Should `package B` be a major bump if `package A` got a major bump? Hard to know for sure.

To be honest, I don't think semantic-release should even try to handle that side of it, and it should be left to the author to manually handle. All I want is the commit filtering work which you've done so packages in a mono repo only bump when their individual code paths have had significant commits.

By the way, I use your package [`semantic-release-plus`](https://www.npmjs.com/package/semantic-release-plus) and it works beautifully. I do worry about its future though with it not being in the main repo.

I'm curious, did you do this work because you also have a mono repo with multiple packages you need to manage?

**Comment by JoA-MoS** [comment]
Yea... I tried to explain that but since it doesn't seem they work with monorepo's it was very hard to communicate what you are saying. 

We use semantic-release-plus inside my national company for our regional public website in a nx repo. Here is an example of adding semantic-release-plus to an nx repo https://github.com/JoA-MoS/nx-jenkins-build/pull/4.

I have been pretty good about merging the upstream changes of semantic-release into semantic-release-plus within a week.
